py/htm/examples/hotgym.py

```diff
@@ -21,7 +21,7 @@
  'sdrc_alpha': 0.1,
  'sp': {'boostStrength': 3.0,
         'columnCount': 1638,
-        'numActiveColumnsPerInhArea': 72,
+        'localAreaDensity': 0.04395604395604396,
```

Comment:
thanks for fixing this, @ctrl-z-9000-times ! 
Btw, can you make some param (steps) to optionally limit the number of steps the code does. 
My aim would be to run the code in the CI, as a unit test, or atleast from the CI file. So we can detect when something breaks. Ideally all runable code would be ran.

Reply:
It's ready now, I tested it on my computer.

> run the code in the CI

That would be good.  I'm not even sure we need to shorten the data set? It runs pretty fast as it is.  It takes less than 1 minute on my computer.

Reply:
That's good to hear. A test would be ideal, the CI script would be quick. Are there some measurable metrics to take from this example? We don't need anymore "exact outputs", but avg anomaly and say last classification would be nice.

Reply:
The example also measures the root-mean-square of the prediction errors, which should be checked in CI.